lib/experimental-i18n-theme.json

```diff
@@ -1,7 +1,6 @@
 {
 	"settings": {
```

Comment:
Why do we have this separate i18n-theme.json file?

Reply:
The idea of this file is to allow WP-CLI to always know what theme.json fields are translatable, this file is the source of truth about what strings should be extracted, so If we change the shape wp-cli does not needs an update.